### PR TITLE
Allow dots in selectors for purgeCss

### PR DESCRIPTION
**Pull request by spiess-demos**
As tailwind uses the dot for selector names, we have to tell that to purgeCss.

### CODE REVIEW

**Review by graupnerdemos — COMMENTED** [review]
What about adding `[`, and `]` while we're at this. Those can also be part of Tailwind selectors.

File: client/fe/webpack/moduleRules.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ if (config.cssPurge.enabled) {
       content: config.cssPurge.paths,
       defaultExtractor (content) {
         const contentWithoutStyleBlocks = content.replace(/<style[^]+?<\/style>/gi, '')
-        return contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/]+/g) || []
+        return contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/.]+/g) || []
```
Comment (salisdemos) [inline client/fe/webpack/moduleRules.js:83]: shouldn't the `.` be escaped?

Comment (graupnerdemos) [reply client/fe/webpack/moduleRules.js:83]: Not when it's in a character group.

Comment (spiess-demos) [reply client/fe/webpack/moduleRules.js:83]: within square brackets, the dot loses its special meaning and is treated as a literal dot.

**Comment by spiess-demos** [comment]
> What about adding `[`, and `]` while we're at this. Those can also be part of Tailwind selectors.

eab1caefbb6eb13a09eb9ae2c20cfeaa207e5258

**Pull request merged**